### DISCUSSION TITLE
what is the default user name and password

### DISCUSSION THREAD

**Discussion by AnythingLinux** [appwrite/appwrite#5101] [Q&A] [answered]
I have deploy appwrite on localhost now looking for default user name and password for login or how to login appwrite?
![Screenshot from 2023-02-10 20-48-49](https://user-images.githubusercontent.com/1058663/218122087-d26c9ee2-1c14-41aa-b0e2-6f0ea471fbf9.png)

**Comment by Yavnik** [comment, marked as answer]
After installing Appwrite for the first time, you'll have to create a root account.

Go to the sign up page and create a new account.

**Reply by jai-dewani** [reply]
While creating an account, I am getting this error that says "Server Error" but I am unable to find out how to check what could be the issue

![image](https://github.com/appwrite/appwrite/assets/33520257/92f005f5-bbcd-4928-8e65-e296f60d991c)

**Comment by LeiYangGH** [comment]
isn't there any 'admin' account by default? how to manage other users' info in the web pages?

**Reply by DaoShen** [reply]
Default admin accounts are a scary attack vector to leave open. Is that really needed?

**Reply by LeiYangGH** [reply]
if not needed, how does appwrite cloud manage users(developers)?

**Reply by DaoShen** [reply]
One of the team members responded on the ticket here. Admin users are created as part of installing: https://github.com/appwrite/appwrite/discussions/5101#discussioncomment-4937456
